subgraph/subgraph.yaml

```diff
@@ -45,7 +45,7 @@ dataSources:
               - name: MarketContractProxy
                 file: ../build/contracts/MarketContractProxy.json
           eventHandlers:
-              - event: PositionTokensMinted(indexed uint256,string,indexed address,address,indexed address,address,uint256,address,address,address,uint256)
+              - event: PositionTokensMinted(uint256,indexed uint256,string,indexed address,address,indexed address,address,address,address,address,uint256)
                 handler: handlePositionTokensMinted
```

Comment:
Needed to update the graph as the event was modified.

Reply:
@andrei-anisimov this is the only update needed, right?